template/fr.opensagres.xdocreport.template/src/main/java/fr/opensagres/xdocreport/template/formatter/FieldsMetadataXMLSerializer.java

```diff
@@ -82,6 +82,7 @@ public FieldsMetadata load( Reader input )
         XMLReader saxReader = XMLReaderFactory.createXMLReader();
         FieldsMetadataContentHandler myContentHandler = new FieldsMetadataContentHandler();
         saxReader.setContentHandler( myContentHandler );
+        
```

Comment:
Here you should add some piece of code which looks like this

...
SAXParserFactory factory = SAXParserFactory.newInstance();
SAXParser saxParser = factory.newSAXParser();

factory.setFeature("https://xml.org/sax/features/external-general-entities", false);
saxParser.getXMLReader().setFeature("https://xml.org/sax/features/external-general-entities", false);
factory.setFeature("https://apache.org/xml/features/disallow-doctype-decl", true); 
...